test/strategy.test.js

```diff
@@ -155,6 +155,30 @@ describe('auth0 strategy', function () {
       should.not.exist(extraParams.acr_values);
     });
 
+    it('should map additional_params fields', function () {
+      var extraParams = this.strategy.authorizationParams({
+        additional_params: {
+          test_param: 'dummy:1',
+        }
+      });
+      extraParams.test_param.should.eql('dummy:1');
+    });
+
+    it('should override explicit fields set in additional_params', function () {
+      var extraParams = this.strategy.authorizationParams({
+        additional_params: {
+          acr_values: 2,
+        },
+        acr_values: 1
+      });
+      extraParams.acr_values.should.eql(2);
```

Comment:
```suggestion
      extraParams.acr_values.should.eql(1);
```

This test should be testing that additional_params don't override the explicit fields, however it is testing that additional_params is overriding the explicit.  Additionally, `acr_values` needs to be a string for it to be excepted, you can see an example of an acr value at https://auth0.com/docs/mfa/guides/configure-step-up-web-apps ie: `http://schemas.openid.net/pape/policies/2007/06/multi-factor`